Userland/Libraries/LibJS/Runtime/Temporal/TimeZone.cpp

```diff
@@ -454,21 +454,24 @@ ThrowCompletionOr<Object*> to_temporal_time_zone(GlobalObject& global_object, Va
 
     // 4. If parseResult.[[Name]] is not undefined, then
     if (parse_result.name.has_value()) {
-        // a. If ParseText(StringToCodePoints(parseResult.[[Name]], TimeZoneNumericUTCOffset)) is not a List of errors, then
-        if (is_valid_time_zone_numeric_utc_offset_syntax(*parse_result.name)) {
-            // i. If parseResult.[[OffsetString]] is not undefined, and ! ParseTimeZoneOffsetString(parseResult.[[OffsetString]]) ≠ ! ParseTimeZoneOffsetString(parseResult.[[Name]]), throw a RangeError exception.
-            if (parse_result.offset_string.has_value() && (MUST(parse_time_zone_offset_string(global_object, *parse_result.offset_string)) != MUST(parse_time_zone_offset_string(global_object, *parse_result.name))))
-                return vm.throw_completion<RangeError>(global_object, ErrorType::TemporalTimeZoneOffsetStringMismatch, *parse_result.offset_string, *parse_result.name);
+        // a. Let name be parseResult.[[Name]].
+        auto& name = *parse_result.name;
+
+        // b. If ParseText(StringToCodePoints(parseResult.[[Name]], TimeZoneNumericUTCOffset)) is not a List of errors, then
```

Comment:
`parseResult.[[Name]]` => `name` to match https://tc39.es/proposal-temporal/#sec-temporal-totemporaltimezone 4.b